### ISSUE TITLE
How to get thw pcd.npz file when training on KITTI

### ISSUE THREAD

**Issue by sonnefred** [DNMP/DNMP#17]
Hello, thanks for your great work! I'm trying to train a model on KITTI, but the pcd.npz file is required when running the trainer_geometry.py, I wonder how to get it? Thanks a lot.

**Comment by FanLu97** [comment]
Hello,
you can refer to `preprocess/README.md` to generate the point cloud.

**Comment by sonnefred** [comment]
> Hello, you can refer to `preprocess/README.md` to generate the point cloud.

Thanks for your reply, I have read it which said the point clouds should be generated using MetaShape. I wonder whether I can use the point clouds provided in KITTI-360 dataset? What's the reason why you don't use it? Thanks.

**Comment by FanLu97** [comment]
Yes, definitely you can use the point clouds provided in KITTI-360 dataset. We don't use it because we found that some point clouds are incomplete when the height of the scene is too large (the heights of LiDAR point clouds are limited). However, you can use the point clouds in KITTI-360 dataset for your convenience.

**Comment by sonnefred** [comment]
> Yes, definitely you can use the point clouds provided in KITTI-360 dataset. We don't use it because we found that some point clouds are incomplete when the height of the scene is too large (the heights of LiDAR point clouds are limited). However, you can use the point clouds in KITTI-360 dataset for your convenience.

OK, thanks, and do I need to do any extra process to the point clouds in KITTI-360 or just read it and make it a npz file?

**Comment by FanLu97** [comment]
You may need to merge multiple point clouds according to the image frames and make sure the coordinate system is right (you can project the point cloud onto image plane to check that).

**Comment by sonnefred** [comment]
> You may need to merge multiple point clouds according to the image frames and make sure the coordinate system is right (you can project the point cloud onto image plane to check that).

OK, thanks, will try it.

**Comment by sonnefred** [comment]
Hi, I noticed that you used colmap to generate depth map and normal map in the pre-process. Is this neccessary? Can I train the model without these two maps? Thanks.

**Comment by jiaangwu** [comment]
> Hello, thanks for your great work! I'm trying to train a model on KITTI, but the pcd.npz file is required when running the trainer_geometry.py, I wonder how to get it? Thanks a lot.


How did you obtain the point cloud data? Can you share it?
